j5/backends/backend.py

```diff
@@ -68,7 +68,7 @@ def __new__(mcs, name, bases, namespace, **kwargs):  # type:ignore
             return cls
 
         # Check if this is an abstract Backend.
-        if len(cls.__bases__) == 1 and cls.__base__.__name__ != "Backend":
+        if hasattr(cls, "ABSTRACT") and cls.ABSTRACT:
```

Comment:
```
>>> class Abstract(metaclass=ABCMeta):
...     @abstractmethod
...     def my_method(self):
...             pass
... 
>>> class NotAbstract(Abstract):
...     def my_method(self):
...             pass
... 
>>> bool(Abstract.__abstractmethods__)
True
>>> bool(NotAbstract.__abstractmethods__)
False
```

Reply:
d9c79d4

Have tested and it appears to work! :+1: